src/components/dateRange.jsx

```diff
@@ -4,6 +4,7 @@ import moment from 'moment';
 import React from 'react';
 import DayPicker, { DateUtils } from 'react-day-picker';
 import { Button, Col, ButtonGroup, Row } from 'react-bootstrap';
+import 'react-day-picker/lib/style.css';
```

Comment:
could the following be update or replaced https://github.com/codefordc/dc-campaign-finance-watch/blob/develop/src/styles/reactdaypicker.css?

could the component style change move to the style dir?